File: app/src/main/java/com/upclicks/ffcapp/models/requests/LoginRequest.java
package com.upclicks.ffcapp.models.requests;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class LoginRequest {
    @SerializedName("UsernameOrEmailAddress")
    @Expose
    private String usernameOrEmailAddress;
    @SerializedName("Password")
    @Expose
    private String password;

    public void setUsernameOrEmailAddress(String usernameOrEmailAddress) {
        this.usernameOrEmailAddress = usernameOrEmailAddress;
    }

    public void setPassword(String password) {
        this.password = password;
    }
}

File: app/src/main/java/com/upclicks/ffc/ui/authentication/model/request/LoginRequest.kt
package com.upclicks.ffc.ui.authentication.model.request

import com.google.gson.annotations.Expose

import com.google.gson.annotations.SerializedName


class LoginRequest {
    @SerializedName("UsernameOrEmailAddress")
    @Expose
    var usernameOrEmailAddress: String? = null

    @SerializedName("Password")
    @Expose
    var password: String? = null

    @SerializedName("rememberClient")
    @Expose
    var rememberClient: Boolean? = null

}